tests/Integration/Behaviour/Features/Scenario/Product/duplicate_product.feature

```diff
@@ -0,0 +1,158 @@
+      | redirect_target  | product2                                           |
+    And carrier carrier1 named "ecoCarrier" exists
+    And carrier carrier2 named "Fast carry" exists
+    And I update product product1 shipping information with following values:
+      | width                            | 10.5                                                  |
+      | height                           | 6                                                     |
+      | depth                            | 7                                                     |
+      | weight                           | 0.5                                                   |
+      | additional_shipping_cost         | 12                                                    |
+      | delivery time notes type         | specific                                              |
+      | delivery time in stock notes     | en-US:product in stock ;fr-FR:en stock                |
+      | delivery time out of stock notes | en-US:product out of stock ;fr-FR:En rupture de stock |
+      | carriers                         | [carrier1,carrier2]                                   |
+    And I add new supplier supplier1 with following properties:
+      | name             | my supplier 1            |
+      | address          | Donelaicio st. 1         |
+      | city             | Kaunas                   |
+      | country          | Lithuania                |
+      | enabled          | true                     |
+      | description      | en-US:just a supplier    |
+      | meta title       | en-US:my supplier nr one |
+      | meta description | en-US:                   |
+      | meta keywords    | en-US:sup,1              |
+      | shops            | [shop1]                  |
+    And I set product product1 default supplier to supplier1 and following suppliers:
+      | reference         | supplier reference | product supplier reference     | currency | price tax excluded |
+      | product1supplier1 | supplier1          | my first supplier for product1 | USD      | 10                 |
+    And I set following related products to product product1:
+      | product2 |
+    And I update product product1 with following customization fields:
+      | reference    | type | name                                                                       | is required |
+      | customField1 | text | en-US:text on top of left lense ;fr-FR:texte en haut de la lentille gauche | true        |
+    And I add new attachment "att1" with following properties:
+      | description | en-US:puffin photo nr1 ;fr-FR:macareux |
+      | name        | en-US:puffin ;fr-FR:macareux           |
+      | file_name   | app_icon.png                           |
+    And I associate attachment "att1" with product product1
+#todo: add specific prices & priorities, test combinations, packs
+    When I duplicate product product1 to a copy_of_product1
+    Then product "product1" should have following values:
```

Comment:
:thumbs 
and copy_of_product1 should actually be deactivated.

Reply:
I just realized we don't yet have a command to activate the product :smile:

Reply:
Do you think we should implement dedicated command, or add "active" in one of other commands like `UpdateBasicInformation` or `UpdateOptions`?

Reply:
mistaken.
There is already `UpdateProductStatusCommand` which should be good to go

Reply:
Yep, and you still haven't fixed the checked product You are checking the initial one (product1) instead of the duplicate one (copy_of_product1) You could activate product1 in your background init Then in the tests you can check both that product1 is still active (didn't change because it was copied) but copy_of_product1 has been duplicated but active is false

Reply:
sadly its a blocker. I'll have to add tests for UpdateStatusCommand (in another PR), to be able to use it here.